Evaluation/Input/evaluation_plan.json

```diff
@@ -579,7 +581,7 @@
                     },
                     {
                         "Caption": "Itraconazole po solution fed",
-                        "Symbol": "Cross",
+                        "Symbol": "Hexagon",
```

Comment:
@Yuri05 Second hexagon - different group

Reply:
yep, you are right, they belong to the different plots. My bad.